practice-one/src/styles/components/footer.css

```diff
@@ -0,0 +1,91 @@
+.footer-container {
+  margin-top: 158px;
+}
+
+.website-summary-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.brief-description {
+  width: 325px;
+  padding-left: 56px;
+  margin-right: 109px;
+}
+
+.brief-heading {
+  color: var(--color-primary,#F54748);
+  font-weight: var(--font-weight-sub-heading, 600);
+  line-height: var(--line-height-normal, 1.3);
+  font-size: var(--font-size-extra-large, 32px);
```

Comment:
recheck for all

Reply:
I think value after (,) is unecessary but when i check it in css validator it displayed warning. Can you make clear this issue 
![image](https://github.com/user-attachments/assets/e0ecb42b-92c7-48db-b782-04e8810fdbb3)